contrib/pyln-proto/pyln/proto/zbase32.py

```diff
@@ -34,6 +35,7 @@ def _message_to_bitarray(message):
 
 
 def _bitarray_to_message(barr):
+    """Decodes a bitarray with length multiple of 5 to a byte message (removing the trailing zeros if necessary)."""
```

Comment:
Should the trimming assert that the trimmed part is all 0s?

Reply:
I was assuming the input is zbase32 encoded, in which case it must be. Given the function is supposed to be internal I didn't care about checking the input for correctness, but I can add the check and except if you think it's worth having it.

Reply:
Maks sense.